pkg/controller/multi/controller.go

```diff
@@ -131,6 +131,7 @@ func (c *Controller) mutatePodSpec(namespace, specPath string, pod corev1.PodSpe
 				// convert digest to patch
 				glog.Infof("Mutation #: %s %d  Image name: %s", containerType, containerIndex, img.String())
 				if strings.Contains(container.Image, img.String()) {
+					// TODO: seems odd to have both tag and digest
```

Comment:
TODO here

Reply:
it is a TODO and should be investigated, it is not a change from master, would an issue be better?

Reply:
made an issue https://github.com/IBM/portieris/issues/90